testnet/stacks-node/src/config.rs

```diff
@@ -14,8 +14,11 @@ use stacks::util::secp256k1::Secp256k1PublicKey;
 use stacks::vm::costs::ExecutionCost;
 use stacks::vm::types::{AssetIdentifier, PrincipalData, QualifiedContractIdentifier};
 
-use super::neon_node::TESTNET_PEER_VERSION;
-use super::node::TESTNET_CHAIN_ID;
+pub const TESTNET_CHAIN_ID: u32 = 0x80000000;
+pub const TESTNET_PEER_VERSION: u32 = 0xfacade01;
+
+pub const MAINNET_CHAIN_ID: u32 = 0x00000001;
+pub const MAINNET_PEER_VERSION: u32 = 0xfacade01;
```

Comment:
This should be the node's semantic version, e.g. `0x18000000` (represting `24.0.0.0`).  The software already treats the highest byte in the version as a major version number, and considers two version values equal if their highest bytes match for the purposes of hashing and equality (e.g. `0x18000000` == `0x18ffffff`).

Reply:
Address with https://github.com/blockstack/stacks-blockchain/pull/2173/commits/a7c78d4e28d380466449a8734056b0d06844f733

Speaking of these `24.0.0.0` versions (also used for the DB versions), should we do a pass and reset to `1.0.0.0`, or `2.0.0.0`?